src/main/java/uk/gov/hmcts/divorce/common/model/CaseData.java

```diff
@@ -735,6 +735,16 @@ public Integer getPaymentTotal() {
             .reduce(0, Integer::sum);
     }
 
+    @JsonIgnore
+    public Boolean wasLastPaymentUnsuccessful() {
+        return payments != null && payments
+            .stream()
+            .reduce((first, second) -> second)
```

Comment:
it would be more useful if you named the variables id, payment

Reply:
Technically this is the current and previous item in the stream, I'll update this to reflect 👍